senders/pagerduty/init.go

```diff
@@ -1,22 +1,28 @@
 package pagerduty
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/moira-alert/moira"
 )
 
 // Sender implements moira sender interface for pagerduty
 type Sender struct {
-	ImageStore moira.ImageStore
-	logger     moira.Logger
-	frontURI   string
-	location   *time.Location
+	ImageStores  map[string]moira.ImageStore
+	imageStoreID string
+	logger       moira.Logger
+	frontURI     string
+	location     *time.Location
 }
 
 // Init loads yaml config, configures the pagerduty client
 func (sender *Sender) Init(senderSettings map[string]string, logger moira.Logger, location *time.Location, dateTimeFormat string) error {
 	sender.frontURI = senderSettings["front_uri"]
+	sender.imageStoreID = senderSettings["image_store"]
+	if sender.imageStoreID == "" {
```

Comment:
Let's just not save the picture if the type is not put. I think not every user wants to buy s3, but they will want to use pagerduty. Then with the release I will add this to the documentation.